warehouse/query-core/src/main/java/datawave/query/transformer/KeywordCloudTransformer.java

```diff
@@ -0,0 +1,21 @@
+package datawave.query.transformer;
+
+import java.util.List;
+
+import datawave.marking.MarkingFunctions;
+import datawave.microservice.query.Query;
+import datawave.query.tables.keyword.KeywordQueryState;
+import datawave.webservice.query.result.event.ResponseObjectFactory;
+import datawave.webservice.result.BaseQueryResponse;
+
+public class KeywordCloudTransformer extends KeywordQueryTransformer {
```

Comment:
Is this used anywhere or in any way different than the KeywordQueryTransformer?

Reply:
Created it and then I didn't end up using it, so I'll remove this.